src/libstd/sys/unix/process/process_common.rs

```diff
@@ -195,6 +193,33 @@ impl Command {
         self.gid = Some(id);
     }
 
+    pub fn saw_nul(&self) -> bool {
+        self.saw_nul
+    }
+    pub fn get_envp(&self) -> &Option<Vec<*const c_char>> {
+        &self.envp
+    }
+    pub fn get_argv(&self) -> &Vec<*const c_char> {
+        &self.argv
+    }
+
+    #[cfg(not(target_os="fuchsia"))]
+    pub fn get_cwd(&self) -> &Option<CString> {
+        &self.cwd
+    }
+    #[cfg(not(target_os="fuchsia"))]
+    pub fn get_uid(&self) -> Option<uid_t> {
+        self.uid
+    }
+    #[cfg(not(target_os="fuchsia"))]
+    pub fn get_gid(&self) -> Option<gid_t> {
+        self.gid
+    }
```

Comment:
These are #[cfg]'d out for lints I think, right? In that case could these just be tagged with `#[allow(dead_code)]`?